README.md

```diff
@@ -1,2 +1,55 @@
 # microdata-crypt
-A repository to encrypt and decrypt dataset files.
+A repository to encrypt and decrypt dataset files based on Python cryptography package.
+
```

Comment:
Nice to add how to install this extra package.

Reply:
Done!